SUMMARY.md

```diff
@@ -9,4 +9,5 @@
   - [Roles](roles.md)
   - [Working Open](working-open.md)
 * [Guides](guides.md)
+  - [Projects](projects.md)
```

Comment:
I don't really know where this should fit. Is it a guide or a subheading under `Our Virtual Office`?

Reply:
:+1: Subsection of guide feels right

Reply:
But it's weird because no other guide is a subsection in the `SUMMARY.md` just one huge index.

Reply:
Ok I removed it from `SUMMARY.md` and added as a Guide, based on what I think you mean.